src/Tools/RepoUtil/VerifyCommand.cs

```diff
@@ -74,7 +74,7 @@ private bool VerifyRepoConfig(TextWriter writer)
             var packages = ProjectJsonUtil
                 .GetProjectJsonFiles(_sourcesPath)
                 .SelectMany(x => ProjectJsonUtil.GetDependencies(x));
-            var set = new HashSet<NuGetPackage>(packages);
+            var set = new HashSet<NuGetPackage>(packages, default(Constants.IgnoreGenerateNameComparer));
```

Comment:
Re: Using `default(...)` for the comparer instead of (the more usual) ".Instance" or ".Default" static property/field. I haven't seen this before. The comparer is a struct, so this is a boxed allocation, right?